code/modules/hydroponics/grown_predefined.dm

```diff
@@ -6,6 +6,9 @@
 /obj/item/weapon/reagent_containers/food/snacks/grown/ambrosiavulgaris
 	plantname = "ambrosia"
 
+/obj/item/weapon/reagent_containers/food/snacks/grown/potato
+	plantname = "potato"
+
```

Comment:
Other possibly useful fruits that lack an entry here:
* Nettles
* Golden apples
* Wheat
* Bluespace tomatoes
* Money pods
* Nova flowers